src/util.js

```diff
@@ -268,6 +272,33 @@ define([
         pos = pos + text.length;
         that.setCaretPosition(ctrl, start, pos);
     };
+
+
+    that.xmlDiff = function (localForm, serverForm, opts) {
```

Comment:
slimmed down version of https://github.com/dimagi/Vellum/blob/master/tests/utils.js#L28

Slimmed it down because I don't think color will work right now. Would have to do a bit of work or use some pretty diff coloring library.